lib/buffer.js

```diff
@@ -1201,35 +1197,80 @@ Buffer.prototype.readInt32BE = function readInt32BE(offset, noAssert) {
 };
 
 
-Buffer.prototype.readFloatLE = function readFloatLE(offset, noAssert) {
-  offset = offset >>> 0;
-  if (!noAssert)
-    checkOffset(offset, 4, this.length);
-  return _readFloatLE(this, offset);
+// For the casual reader who has not at the current time memorized the
+// IEEE-754 standard in full detail: floating point numbers consist of
```

Comment:
Ah yes, IEEE-754... Lite-afternoon reading that goes perfect with coffee and scones.